inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -36,29 +37,9 @@
 #include "precision_utils.h"
 #include <ie_plugin_config.hpp>
 
-#include "utils/blob_dump.h"
 #include "utils/general_utils.h"
-
-/*****************************************************
- * Debug capability
- *  - BLOB_DUMP_PATH : Specify with existing folder name
- *    to dump intermediate blobs into it
- *  - PRINT_GRAPH_INFO : Define it to enable printing
- *    additional information to std output.
- *
- *****************************************************/
-// #define BLOB_DUMP_PATH "mkldnn_dump"
-// #define PRINT_GRAPH_INFO
```

Comment:
How we can enable this macro in this new paradigm?  `PRINT_GRAPH_INFO` is something different from blobs dumping.

Reply:
Yes, wanted to discuss it. It seems logical to have it under environment variable as well, but the we should isolate reading of environment variables in some separate class, pass it to node dumper and "graph_printer", etc

Reply:
So may be it makes sense to leave `PRINT_GRAPH_INFO` as is until we find a better solution?

Reply:
Discuss, PRINT_GRAPH_INFO will be kept